web/packages/teleport/src/components/HeadlessRequestDialog/HeadlessRequestDialog.tsx

```diff
@@ -25,39 +25,75 @@ import Dialog, {
 import { Danger } from 'design/Alert';
 import { ButtonPrimary, ButtonSecondary, Text } from 'design';
 
-export default function HeadlessSsoDialog({
+export default function HeadlessRequestDialog({
   ipAddress,
   onAccept,
   onReject,
   errorText,
 }: Props) {
+  const dialogContent = () => {
+    const getText = (errorText: string) => {
+      if (errorText) {
+        return (
+          <Text textAlign="center">
+            The requested session doesn't exist or is invalid. Please generate a
+            new request.
+            <br />
+            <br />
+            You can close this window.
+          </Text>
+        );
+      }
+
+      return (
+        <Text textAlign="center">
+          Someone has initiated a command from {ipAddress}. If it was not you,
+          click reject and contact your administrator.
```

Comment:
```suggestion
          click Reject and contact your administrator.
```

From what I see this is the capitalization we use in docs. Grepping the project for `"\bclick "` didn't really return any results from the web folder.

Reply:
Added